helm/cloud-provider-cloud-director/values.yaml

```diff
@@ -11,10 +11,11 @@ global:
     vipSubnet: ""  # Virtual IP CIDR for the external network
     clusterid: ""  # "Infra Id" field of VCDCluster object in management cluster
     vAppName: ""
-    enableVirtualServiceSharedIP: false  # supported for VCD >= 10.4 - not used yet
+    enableVirtualServiceSharedIP: false  # Multiple VS can share an IP if true.
```

Comment:
what is VS? Virtual service?

Reply:
![image](https://user-images.githubusercontent.com/19745768/220905531-d67b6d57-f24d-4971-91d2-727744570ecf.png)